File: humanReadableTime/kata.js

function humanReadable(seconds) {
    const hours = Math.floor(seconds / 60 / 60);
    const mins = Math.floor(seconds / 60) % 60;
    seconds = seconds % 60;
    return `${hours >= 10 ? hours : '0' + hours}:${mins >= 10 ? mins : '0' + mins}:${seconds >= 10 ? seconds : '0' + seconds}`;
}


console.log(humanReadable(59));

File: humanReadableTime/kata.py
import math

def make_readable(seconds: int):
    hours = math.floor(seconds / 60 / 60)
    mins = math.floor(seconds / 60) % 60
    return '{}:{}:{}'.format(str(hours) if hours >= 10 else "0" + str(hours), str(mins) if mins >= 10 else "0" + str(mins), str(seconds % 60)
                             if (seconds % 60) >= 10 else "0" + str(seconds % 60))


print(make_readable(60))
